docker-compose.yaml

```diff
@@ -14,20 +19,26 @@ services:
     build: .
     container_name: chainpoint-node
     ports:
-      - "80:8080"
+      - '${PORT}:${CHAINPOINT_NODE_PORT}'
     networks:
       - chainpoint-node
+      - boltbox-simnet
+      - chainpoint-core_chainpoint
     environment:
       HOME: /root
       HOT_WALLET_PASS: ${HOT_WALLET_PASS}
       LND_SOCKET: ${LND_SOCKET}
-      CHAINPOINT_CORE_CONNECT_IP_LIST: "${CHAINPOINT_CORE_CONNECT_IP_LIST}"
-      AGGREGATION_INTERVAL_SECONDS: "${AGGREGATION_INTERVAL_SECONDS}"
-      PROOF_EXPIRE_MINUTES: "${PROOF_EXPIRE_MINUTES}"
-      POST_HASHES_MAX: "${POST_HASHES_MAX}"
-      POST_VERIFY_PROOFS_MAX: "${POST_VERIFY_PROOFS_MAX}"
-      GET_PROOFS_MAX: "${GET_PROOFS_MAX}"
-      MAX_SATOSHI_PER_HASH: "${MAX_SATOSHI_PER_HASH}"
+      LND_MACAROON: ${LND_MACAROON}
+      LND_TLS_CERT: ${LND_TLS_CERT}
```

Comment:
Added to support both raw values as well as directories, also useful for local dev when you want to use an existing lightning node that is connected, synced, and funded but data dir is not mounted or easily mounted.